src/axolotl/monkeypatch/accelerate/fsdp2.py

```
@@ -254,8 +254,10 @@ def fsdp2_prepare_model(accelerator, model: torch.nn.Module) -> torch.nn.Module:
         "offload_policy": fsdp2_plugin.cpu_offload,
         # `fully_shard` doesn't accept `None` in case of `MixedPrecisionPolicy`
         "mp_policy": fsdp2_plugin.mixed_precision_policy or MixedPrecisionPolicy(),
+        "mesh": accelerator.state.device_mesh[accelerator.state.parallelism_config.model_shard_dim_names]
     }
-
+    if accelerator.state.device_mesh and accelerator.state.parallelism_config:
+        fsdp2_kwargs
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)